NineChronicles.Standalone/Properties/NineChroniclesNodeServiceProperties.cs

```diff
@@ -13,6 +13,10 @@ namespace NineChronicles.Standalone.Properties
 {
     public class NineChroniclesNodeServiceProperties
     {
+        // A private key that is used in mining and signing transactions, which is different
+        // with the private key used in swarm to sign messages.
+        public PrivateKey PrivateKey { get; set; }
```

Comment:
Since we decided to use its option name as `--miner-private-key`, how about renaming this also like `MiningPrivateKey`?

Reply:
Amended this too.